tests/test_client.py

```diff
@@ -0,0 +1,330 @@
+import asyncio
+
+import pytest
+from aioresponses import CallbackResult, aioresponses
+
+from flexmeasures_client.client import FlexMeasuresClient
+
+
+def test__init__():
+    flexmeasures_localhost = FlexMeasuresClient("password", "email")
+    assert flexmeasures_localhost.__dict__ == {
+        "password": "password",
+        "email": "email",
+        "access_token": None,
+        "host": "localhost:5000",
+        "scheme": "http",
+        "ssl": False,
+        "api_version": "v3_0",
```

Comment:
Let's test the default by not passing this explicitly.

Reply:
Was this supposed to have been resolved?

Reply:
Somehow my comment is gone. This is the asserted dict not the variables passed into the function. There are 3 fm_client objects being created and tested against the 3 dicts. It can be refactored to make it cleaner: #16